core/modules/views_ui/views_ui.admin.inc

```diff
@@ -4401,21 +4403,21 @@ function views_ui_admin_settings_basic() {
     }
   }
 
-  // This is not currently a fieldset but we may want it to be later,
-  // so this will make it easier to change if we do.
+  // This is not currently a fieldset but we may want it to be later, so this
+  // will make it easier to change if we do.
   $form['basic'] = array();
 
   $form['basic']['show_master_display'] = array(
     '#type' => 'checkbox',
-    '#title' => t('Always show the master display'),
-    '#description' => t('Advanced users of views may choose to see the master (i.e. default) display.'),
+    '#title' => t('Always show the <em>Master</em> display'),
+    '#description' => t('Show the <em>Master</em> (i.e. default) display, which is otherwise hidden by default.'),
     '#default_value' => $config->get('show_master_display'),
   );
 
   $form['basic']['display_embed'] = array(
     '#type' => 'checkbox',
-    '#title' => t('Show the embed display in the ui.'),
-    '#description' => t('Allow advanced user to use the embed view display. The plugin itself works if it\'s not visible in the ui'),
+    '#title' => t('Allow <em>embed</em> displays to be created via the UI'),
+    '#description' => t('This enables another type of views display called <em>embed</em>, which can be added via the "+ Add" button when creating/editing a view. This type of display can be used in code via <a href="@api_link"><code>views_embed_view()</code></a>. The plugin for <em>emebed</em> displays itself works even if this setting is disabled.', array('@api_link' => url('https://api.backdropcms.org/api/backdrop/core%21modules%21views%21views.module/function/views_embed_view'))),
```

Comment:
Let's avoid using words like "Plugin" in the UI. 

`Enables the <em>embed</em> type of display. Embed displays can be used in code via <a href="@api_link"><code>views_embed_view()</code></a>. Embedded displays will still work even if this setting is disabled.'`

Reply:
Great suggestion 👍 ...done!